pkg/services/ngalert/ngalert.go

```diff
@@ -279,6 +279,7 @@ func (ng *AlertNG) init() error {
 	cfg := state.ManagerCfg{
 		Metrics:                        ng.Metrics.GetStateMetrics(),
 		ExternalURL:                    appUrl,
+		ExecuteAlerts:                  ng.Cfg.UnifiedAlerting.ExecuteAlerts,
```

Comment:
Can we invert this to `DisableExecution:`?

I'm concerned about false being the default, and us forgetting to set this value in the ruler.

Reply:
Yeah we could do that, agree 👍